app/helpers/consultation_helper.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+module ConsultationHelper
+  def neighbour_letter_content(consultation)
+    "# #{consultation.neighbour_letter_header}\n\n#{consultation.neighbour_letter_content.gsub('CURRENT_USER',
+                                                                                               current_user.name)}"
```

Comment:
Can't use the current_user helper in models, so have come up with this way round. Happy if someone has a better suggestion